pkg/pool-cow/test/foundry/CowRouter.t.sol

```diff
@@ -604,6 +604,110 @@ contract CowRouterTest is BaseCowTest {
         assertEq(cowRouter.getProtocolFeePercentage(), newProtocolFeePercentage, "Protocol Fee Percentage is not set");
     }
 
+    /********************************************************
+                          getFeeSweeper()
+    ********************************************************/
+    function testGetFeeSweeper() public {
```

Comment:
```suggestion
                          Fee Sweeper
    ********************************************************/
    function testGetFeeSweeper() public {
```
Even in tests, there should be a blank line after section headers (here and elsewhere). I'd just say "Fee Sweeper" here if it's a header; we don't need headers for every function, just every section.